tests/queries/0_stateless/02946_materialize_column_must_not_override_past_values.sql

```diff
@@ -0,0 +1,53 @@
+SET mutations_sync = 2;
+
+SELECT '-- Compact parts';
+
+CREATE TABLE tab (id Int64, dflt Int64 DEFAULT 54321) ENGINE MergeTree ORDER BY id;
```

Comment:
It would be nice to test also MATERIALIZED expressions (e.g. as columns `dflt_default` and `dflt_materialized`).

Reply:
I will add a test for it.

Reply:
Done